docs/package.json

```diff
@@ -3,13 +3,21 @@
 	"version": "0.0.0",
 	"private": true,
 	"scripts": {
+		"build": "robo build",
+		"deploy": "robo deploy",
+		"dev": "docusaurus serve --dir ./public/ && robox dev",
+		"doctor": "sage doctor",
+		"invite": "robo invite",
+		"lint:style": "prettier --write .",
+		"start": "docusaurus build --out-dir ./public/ && robo build && robo start",
```

Comment:
Please do not include builds in `start` commands. This causes a duplicate build because of the step prior.